PluginsAndFeatures/azure-toolkit-for-intellij/src/main/resources/com/microsoft/azure/toolkit/operation/title/sqlserver.properties

```diff
@@ -1,6 +1,5 @@
-sqlserver|server.create=create SQL Server({0}) in subscription({1})
-sqlserver|server.delete=delete SQL Server({0})
-sqlserver|server.open_portal=open configuration page of SQL Server({0}) in portal
-sqlserver|server.show_properties=show properties of SQL Server({0})
-sqlserver|server.open_by_database_tools=open SQL Server({0}) by Database Tools plugin.
-sqlserver|server.database_load|subscription=list all SQL Server databases in server({0}) of subscription({1})
+sqlserver.server.create.task=create SQL Server({0}) in subscription({1})
+sqlserver.server.open_portal=open configuration page of SQL Server({0}) in portal
```

Comment:
no need to change this  since `sqlserver|server` is the service

Reply:
sqlserver|server.create.task

Reply:
updated